### ISSUE TITLE
Ability to lint from standard input

### ISSUE THREAD

**Issue by krzysztof-magosa** [adrienverge/yamllint#47]
Hi,
It would be great if `yamllint` could lint YAML coming from standard input.
It would be helpful when creating editors' integration etc.
Cheers.

**Comment by ssbarnea** [comment]
Indeed this functionality is key as all serious linters support it, allowing easy integration with editors.

**Comment by allanlewis** [comment]
My current solution uses Make like this:
```make
out.yaml:
	some-command-that-outputs-yaml > $@

.INTERMEDIATE: out.yaml

check: out.yaml
	yamllint -s $<

.PHONY: check
```

**Comment by adrienverge** [comment]
This is resolved in version 1.15.0, see https://github.com/adrienverge/yamllint/blob/master/CHANGELOG.rst#1150-2019-02-11.
